### PR TITLE
[Beta] Task 16030248: Dynamic config for AILight

### PR DESCRIPTION
**Pull request by Karlie-777**

### CODE REVIEW

File: AISKULight/src/index.ts [hunk 1]
```diff
@@ -61,12 +64,37 @@ export class ApplicationInsights {
...
+                _addUnloadHook(onConfigChange(cfgHandler, () => {
+                    if (_config.connectionString) {
+                        const cs = parseConnectionString(_config.connectionString);
+                        const ingest = cs.ingestionendpoint;
+                        _config.endpointUrl = ingest ? (ingest + DEFAULT_BREEZE_PATH) : _config.endpointUrl; // only add /v2/track when from connectionstring
+                        _config.instrumentationKey = cs.instrumentationkey || _config.instrumentationKey;
+                    }
+                }));
+                
+                const extensions = [];
```
Comment (MSNev) [inline AISKULight/src/index.ts:84]: `extensions` doesn't look like it's used.

File: BETA_BREAKING_CHANGES.md [hunk 2]
```diff
@@ -133,6 +133,12 @@ The previously exported namespaced globals have been replaced with a reduced set
 
 ### AISKU Light
 
+#### V1 API Changes
```
Comment (MSNev) [inline BETA_BREAKING_CHANGES.md:136]: V2 API Changes :smile:

File: AISKULight/src/index.ts [hunk 3]
```diff
@@ -61,12 +57,26 @@ export class ApplicationInsights {
...
+                "addUnloadCb",
+                "onCfgChange"
             ]);
 
             function _initialize(): void {
+                let cfgHandler: IDynamicConfigHandler<IConfiguration & IConfig> = createDynamicConfig(config || ({} as any), defaultConfigValues);
+                _config = cfgHandler.cfg;
+    
+                core.addUnloadHook(onConfigChange(cfgHandler, () => {
+                    console.log(_config.connectionString)
```
Comment (MSNev) [inline AISKULight/src/index.ts:69]: Probably can remove this now 😄

Comment (Karlie-777) [reply AISKULight/src/index.ts:69]: I Forgot! Thanks Nev!

**Review by MSNev — APPROVED** [review]

**Pull request merged**